src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.exceptions;
+
+public class BookIdMismatchException extends RuntimeException {
```

Comment:
Check out this annotation: `@ResponseStatus(code = HttpStatus. ...  , reason = "...")` that can be useful for your custom exceptions :D